js/Source/Slider.Modify/Slider.Modify.js

```diff
@@ -75,9 +76,22 @@ Slider.Modify = new Class({
     }, this);
   },
 
+  addMoveClass: function(){
+    if(this.options.moveClass && this.options.moveClassTargets){
+      this.options.moveClassTargets.addClass(this.options.moveClass)
+    }
+  },
```

Comment:
to me, it is the same as the code in the 'onComplete' handler. this whole class is very specific to this use case. what i should really do (later) is refactor to make a 'Slider.Fill' class, and then have this specialty class inherit from that.